typescript/orval.config.ts

```diff
@@ -6,6 +6,7 @@ export default defineConfig({
       target: "../openapi.yaml",
     },
     output: {
+      clean: true,
```

Comment:
This will clear out the generated code before each run. This is useful to clear out any stale or broken code.